src/Domain.jl

```diff
@@ -1427,6 +1427,10 @@ export calcthermo
             dydt[d.indexes[1]:d.indexes[2]] .+= inter.y.*inter.F(t)
         elseif isa(inter,Outlet) && d == inter.domain
             dydt[d.indexes[1]:d.indexes[2]] .-= inter.F(t).*ns./N
+        elseif isa(inter,Condensation) && d == inter.domain
```

Comment:
Can I model kinetics in a gas that's over a constant composition liquid with these? I think you have all the parameters for that in these, but this configuration here I think only works if the domain is a liquid domain? 

Do we have a failsafe that warns the user if they try to use one of these new interfaces and it doesn't work with the domain they chose?

Reply:
That's true, I can generalize these interfaces such that it could work for a vapor domain that's over a constant composition liquid. I will add the failsafe warnings. Thanks!

Reply:
I generalize the interface to gas phase domains and add failsafe warning to the interface